setup.py

```diff
@@ -40,9 +40,7 @@ def _dbt_duckdb_version():
         "dbt-core~=1.7.0",
         "duckdb>=0.7.0",
     ],
-    extras_require={
-        "glue": ["boto3", "mypy-boto3-glue"],
-    },
+    extras_require={"glue": ["boto3", "mypy-boto3-glue"], "md": ["duckdb>=0.7.0,<=0.9.2"]},
```

Comment:
I added this so the CI succeeds (MotherDuck currently does not support 0.10).
See run here: https://github.com/motherduckdb/dbt-duckdb/actions/runs/7895023998/job/21546711483

Reply:
No worries